app/models/product_detail.rb

```diff
@@ -4,4 +4,6 @@ class ProductDetail < ApplicationRecord
   belongs_to :product_color
   has_many :order_details, dependent: :destroy
   has_many :orders, through: :order_details
+
+  delegate :product_name, to: :product
```

Comment:
```suggestion
  delegate :name, to: :product, , prefix: :product
```
ri là dc mà, cần j tạo method product_name ở product model cu